### ISSUE TITLE
add feature to announce merged pull requests by MLH fellows

### ISSUE THREAD

**Issue by diananova** [MLH-Fellowship/github-discord-bot#7]

**Comment by gmelodie** [comment]
How are you thinking this could be triggered? Once every X hours, by issuing a command (like `!git shoutout <fellow>` ), or maybe both? You could also have a github event trigger, so when something happens on github you get notified and issue an announcement on discord (that'd be harder to do though).

**Comment by diananova** [comment]
yeah this feature is more complex to implement, so we thought we'd leave for last and see if we have time to implement it.
